src/amo/components/AddonMeta/styles.scss

```diff
@@ -36,3 +36,24 @@
   @include margin-end(-2.5px);
   @include margin-start(-2.5px);
 }
+
+.AddonMeta .MetadataCard-content,
+.AddonMeta .MetadataCard-title {
+  a,
+  a:link,
+  a:visited {
+    color: $grey-50;
```

Comment:
do we want to change the color here for the number too? 

not that this should be a blocker but I am also wondering if there is a more visual way to let the user know it's a link? Maybe this is okay..? But right now unless you hover over it, it's hard to tell..

Reply:
I did this to match the mocks. I think it would be a bit ugly to have the title or the number in blue in among all the grey text, although you make a good point that the fact that it's a link is not obvious.

What do you think @pwalm?